lib/merchant_repository.rb

```diff
@@ -15,48 +15,47 @@ def inspect
 
   def create_merchants(path)
     CSV.foreach(path, headers: true, header_converters: :symbol) do |merchant|
-    @all_merchants << Merchant.new(merchant, self)
-      # require "pry"; binding.pry
+    @all << Merchant.new(merchant, self)
     end
   end
 
   def find_by_id(id)
-    @all_merchants.find do |merchant|
+    @all.find do |merchant|
       merchant.id == id
     end
   end
 
   def find_by_name(name)
-    @all_merchants.find do |merchant|
+    @all.find do |merchant|
       merchant.name.upcase == name.upcase
     end
   end
 
   def find_all_by_name(name)
-    @all_merchants.select do |merchant|
+    @all.select do |merchant|
         merchant.name.downcase.include?name.downcase
     end
   end
 
   def next_highest_merchant_id
-    @all_merchants.max_by do |merchant|
+    @all.max_by do |merchant|
       merchant.id
     end.id + 1
   end
 
   def create(attributes)
     new_merchant = Merchant.new({id: next_highest_merchant_id,
-                                 name: attributes})
-    @all_merchants << new_merchant
+                                 name: attributes}, self)
```

Comment:
Thoughts on making this one line? With only 2 items in the hash, the current formatting catches my eye. Not essential though. If the `next_highest_merchant_id` method name was condensed that could help with the one line formatting.